ui/components/Navigator.js

```diff
@@ -590,7 +590,7 @@ class Navigator extends React.Component {
       return (
         <List disablePadding>
           {children.map(({
-            id, onClickCallback, icon, href, title, children
+            id, onClickCallback, icon, href, title, children, show : showc
```

Comment:
How was then navigator working before? I am just curious.

Reply:
There was no condition when `show` was undefined (i guess), i checked the other places `showc` was used. I also wanted to ask you about it